src/jsx/repo.jsx

```diff
@@ -292,3 +301,26 @@ var Repo = React.createClass({
     }
   }
 });
+
```

Comment:
I slightly disagree. Ultimately, it comes down to ease of use. The major downside of using lodash is that you have to include another JS file, which affects page load. But this is a minimal downside and not something I really care about with Seer.

lodash is better than the built-in methods in several ways:
  (1) It is way more cross-platform than the built-ins ever will be. Certain built-in methods differ across browsers in unexpected ways, but lodash ensures consistent contracts.
  (2) It handles "invalid" inputs like `undefined` and `null` gracefully, while most of the built-in methods will throw uncaught exceptions. For example, `foo.trim()` will throw if `foo` is `undefined` while `_.trim(foo)` will handle that case and return a sensible default of `""`. Error handling is much nicer in lodash because so many fewer errors are actually thrown.
  (3) It is straight up faster than a lot of built-in methods. See [the lodash benchmark suite for examples](https://lodash.com/benchmarks).
  (4) It contains helpers that are not even planned to be built into the browser (as far as I know). Things like [`_.get()`](https://lodash.com/docs#get), [`_.set()`](https://lodash.com/docs#set), and [`_.forOwn()`](https://lodash.com/docs#forOwn) are nice shorthands and make code more readable.